cmd/router/main.go

```diff
@@ -109,6 +110,30 @@ func routeStep(nodeName string, graph v1alpha1.InferenceGraphSpec, input []byte,
 		}
 		return executeStep(route, graph, input, headers)
 	}
+	if currentNode.RouterType == v1alpha1.Mirroring {
+		//* check the step len, if step is nil, return input directly
+		if currentNode.Steps == nil {
+			return input, nil
+		}
+		//* create goroutines for mirror service, and ignore response
+		for i := range currentNode.Steps {
+			step := currentNode.Steps[i]
+			var weight int64
+			if step.Weight != nil {
+				weight = *step.Weight
+			} else {
+				weight = 100
+			}
+			r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Can you put some inline comments on how it is working ?

Reply:
update